book/website/ethical-research/ethics-open-source-governance.md

```diff
@@ -0,0 +1,99 @@
+
+(er-ethics-open-source-governance-summary)=
+## Summary
+
+Open source projects are rooted in ethical principles. 
+Sharing the code so that anyone can use it, welcoming contributors and creating communities around specific projects are the core of open source. 
+But open source communities also have the responsibility to ensure they are a safe, inclusive environment, understand who might be affected by their projects or check that their tools are not being misused. 
+Therefore, Open Source Governance Models should embed those ethical considerations within their structure, but the framework to do so is far from well-established. 
+This chapter discusses some of the key points to take into consideration when designing and implementing ethical guidelines in governance models.
+
+
+(er-ethics-open-source-governance-motivation)=
+## Motivation and Background
+
+When setting up open source projects, it is important to pick a structure for organising the community that takes into account ethical considerations 
+The way that your governance model is designed has a significant impact on the ethics, diversity and inclusion in your project, so critically examining what the implied ethics are of a particular structure is a crucial part of the process of selecting one. 
+It can be difficult to know where to start when considering this aspect of open source projects, so this chapter offers some starting points to consider. 
+
+*This chapter is adapted a speedblog from the Collaborations Workshop 2022, authored by  Yo Yehudi,
+Arielle Bennett, Gemma Turon, Declan Bays, Sarah Gibson, Stephan Druskat, Yadira Sanchez, Sophia Batchelor, which is published under a CC-BY license. Read the original article on Software Sustainability Institute's Blog page* 
+
+(er-ethics-open-source-governance-selfreflection)=
+## Ethical Governance Starts with Self-Reflection 
+
+The process of interrogating the ethics of a governance model starts with {ref}`er-self-reflection` on the nature and intention of your project. 
+Take things right back to first principles and ask yourself (and the project team):
+* Why are you doing this project? 
+* Who is the project for?
+* Who creates and maintains it? 
+* How is the project to be used? 
+
+These questions may not always be captured in the initial README, however by scoping the impact and influence of a project before it begins, a team can lay an ethical foundation from the outset. 
+
+For many projects that are community led, this means involving the people that are affected by the project’s outcome in the design, development, and deployment stages.
+True co-creation is an active process of inclusion, and there are many levels of engagement that can be tailored to a projects’ individual needs (see Figure 1. Arnstein's Ladder of Citizen Participation), which a governance model might also need to take into account, for example, how is delegated power dealt with by the project?  
```

Comment:
```suggestion
True co-creation is an active process of inclusion, and there are many levels of engagement that can be tailored to a projects’ individual needs (see the figure - Arnstein's Ladder of Citizen Participation), which a governance model might also need to take into account, for example, how is delegated power dealt with by the project?  
```

Reply:
Figure numbering in the book is maintained for the entire book, so this may be listed automatically with a number (for example 67 of there are 66 other image before this chapter)